src/server/game/Spells/SpellMgr.cpp

```diff
@@ -5944,6 +5944,11 @@ void SpellMgr::LoadSpellInfoCorrections()
         spellInfo->AttributesEx |= SPELL_ATTR1_FINISHING_MOVE_DURATION;
     });
 
+    // Shadowmeld
```

Comment:
that one is wrong. The cooldown is being triggered on start since 3.0.3. Remove it

Reply:
Good catch. Guess I am too used to Classic 😛